osu.Game.Rulesets.Tau/Mods/TauModFlashlight.cs

```diff
@@ -26,7 +26,7 @@ namespace osu.Game.Rulesets.Tau.Mods
 {
     public class TauModFlashlight : TauModFlashlight<TauHitObject>
     {
-        public override double ScoreMultiplier => 1.2;
+        public override double ScoreMultiplier => 1.55;
```

Comment:
```suggestion
        public override double ScoreMultiplier => 1.15;
```

Reply:
sorry, accidentally read as 1.6x instead of 1.16x changing to 1.55x